packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -189,6 +193,16 @@ class WebView extends StatefulWidget {
   ///     * When a navigationDelegate is set HTTP requests do not include the HTTP referer header.
   final NavigationDelegate navigationDelegate;
 
+  /// Invoked when a page has finished loading.
+  ///
+  /// This is invoked only for the main frame.
+  ///
+  /// When [onPageFinished] is invoked, the page being rendered may not be
```

Comment:
Are you sure that it may not be updated yet? (what does updated mean in this context?) if not I'd just omit this part.

Reply:
In the [WebViewClient](https://developer.android.com/reference/android/webkit/WebViewClient#onPageFinished(android.webkit.WebView,%20java.lang.String)) docs this line is included for onPageFinished

```
This method is called only for main frame. When onPageFinished() is called, the rendering picture may not be updated yet.
```

"Picture" being the canvas the WebView is drawn on. I may need to include that this applies to Android. There's no reference to this in iOS.